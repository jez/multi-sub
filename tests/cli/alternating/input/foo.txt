Change this line
Don't change this line
Also change this line
But still not this line
