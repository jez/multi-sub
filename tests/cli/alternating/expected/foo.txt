Change THIS LINE
Don't change this line
Also change THIS LINE
But still not this line
